data/sql/updates/pending_db_world/rev_1629295484385297983.sql

```diff
@@ -0,0 +1,13 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1629295484385297983');
+
+DELETE FROM `gameobject` WHERE (`id` = 179828) AND (`guid` IN (6974, 6976, 6991, 6995, 7003, 7005, 7006, 7009));
```

Comment:
dis file should not be here :D

Reply:
yep, im not sure why git did that, let me just fix that